### PR TITLE
gdal 3.4.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by SMillerDev — DISMISSED** [review]

**Comment by Moisan** [comment]
It seems like `mapserver` also needs a revision bump.

**Comment by carlocab** [comment]
Can't reproduce the `livecheck` and `audit` failures locally. Opened Homebrew/homebrew-test-bot#699.

**Comment by iMichka** [comment]
I opened a PR for mapnik: #91815

**Review by iMichka — APPROVED** [review]

File: Formula/osmcoastline.rb [hunk 1]
```diff
@@ -17,6 +18,7 @@ class Osmcoastline < Formula
   depends_on "gdal"
   depends_on "geos"
   depends_on "libspatialite"
+  depends_on "lz4"
```
Comment (carlocab) [inline Formula/osmcoastline.rb:21]: Why? Is this not just opportunistic linkage?

Comment (chenrui333) [reply Formula/osmcoastline.rb:21]: yeah, was seeing this in the test failure.

>   ==> /opt/homebrew/Cellar/osmcoastline/2.3.1_1/bin/osmcoastline -v -o output.db input.opl
>   dyld: Library not loaded: /opt/homebrew/opt/lz4/lib/liblz4.1.dylib
>     Referenced from: /opt/homebrew/Cellar/osmcoastline/2.3.1_1/bin/osmcoastline
>     Reason: image not found

Comment (chenrui333) [reply Formula/osmcoastline.rb:21]: Let's see if we can remove this in #92172 (my local 12-arm test out fine)

**Pull request closed without merging**